### PR TITLE
Added a parameter to run xUnit tests in parallel

### PR DESCRIPTION
**Pull request by dkloud**

### CODE REVIEW

File: Content/tasks.tests.unit.cake [hunk 1]
```diff
@@ -57,9 +57,11 @@ Sitecore.Tasks.RunServerUnitTestsTask = Task("Unit Tests :: Run Server Tests")
 
         var _openCoverResultsFilePath = new FilePath($"{Sitecore.Parameters.XUnitTestsCoverageOutputDir}/coverage.xml");
 
+        var parallelismOptionArgument = Sitecore.Parameters.XUnitTestsRunInParallel ? ParallelismOption.All : ParallelismOption.None;
+        Information($"Parallelism Option Argument = {parallelismOptionArgument}");
```
Comment (ghost) [inline Content/tasks.tests.unit.cake]: Please remove this info log

Comment (dvarfolomeev04) [reply Content/tasks.tests.unit.cake]: https://cakebuild.net/api/Cake.Common.Tools.XUnit/ParallelismOption/ this enum contains 4 values. I think it makes sense to have all options available, to not change it in the future again

Comment (dkloud) [reply Content/tasks.tests.unit.cake]: > https://cakebuild.net/api/Cake.Common.Tools.XUnit/ParallelismOption/ this enum contains 4 values. I think it makes sense to have all options available, to not change it in the future again

Good mention. Now we can pass any of the 4 arguments as a string. Is it fine?

Comment (dvarfolomeev04) [reply Content/tasks.tests.unit.cake]: Approved (cannot find such a button here)

**Pull request merged**